pkg/converter/target/utils.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 Antrea Authors.
```

Comment:
should we rename the file as common, or helper, rather than utils?

Reply:
Sure, will do that.